infracost-usage-defaults.large.yml

```diff
@@ -415,6 +415,8 @@ resource_type_default_usage:
   #
   # Terraform AzureRM resources
   #
+  azurerm_app_configuration.my_configuration:
+    requests: 50000 # Monthly number of requests which are above teh 200,000 a day free tier. This is the total number of requests made to the service, including replica requests.
```

Comment:
We need to update the contributing docs, but the idea with this file is to target $20/month per cost component so at $0.06/10k  that works out to about 3333333 overage requests.  The .medium should be half of that, and .small half again.